src/lib/list/selection-list.ts

```diff
@@ -140,6 +142,9 @@ export class MatListOption extends _MatListOptionMixinBase
               /** @docs-private */
               @Inject(forwardRef(() => MatSelectionList)) public selectionList: MatSelectionList) {
     super();
+
+    this._parentStateSubscription = this.selectionList._stateChanges
```

Comment:
Add a comment that explains what this is for?

Also, instead of keeping track of the subscription, we've been moving towards a pattern of having a ` _destroyed` subject and using `takeUntil`